salt/modules/ansiblegate.py

```diff
@@ -365,7 +373,201 @@ def playbooks(
     }
     ret = __salt__["cmd.run_all"](**cmd_kwargs)
     log.debug("Ansible Playbook Return: %s", ret)
-    retdata = json.loads(ret["stdout"])
+    try:
+        retdata = json.loads(ret["stdout"])
+    except ValueError:
+        retdata = ret
     if "retcode" in ret:
         __context__["retcode"] = retdata["retcode"] = ret["retcode"]
     return retdata
+
+
+def targets(**kwargs):
```

Comment:
Why `**kwargs` and not explicit keyword arguments?

Reply:
Fixed. Now they are passed explicetely. :+1: